pkg/mcs/resource_manager/server/token_bukets.go

```diff
@@ -30,6 +30,7 @@ const (
 const (
 	defaultReserveRatio    = 0.05
```

Comment:
Now that we've implemented slot, can we make `defaultReserveRatio` bigger?

Reply:
modify to `0.5`